go/adbc/driver/internal/driverbase/connection.go

```diff
@@ -332,6 +360,35 @@ func (b *ConnectionBuilder) Connection() Connection {
 	return conn
 }
 
+func (cnxn *ConnectionImplBase) GetTraceParent() string {
+	return cnxn.traceParent
+}
+
+func (cnxn *ConnectionImplBase) SetTraceParent(traceParent string) {
+	cnxn.traceParent = traceParent
+}
+
+func (cnxn *ConnectionImplBase) StartSpan(
+	ctx context.Context,
+	spanName string,
+	opts ...trace.SpanStartOption,
+) (context.Context, trace.Span) {
+	ctx, _ = maybeAddTraceParent(ctx, cnxn, nil)
+	return cnxn.Tracer.Start(ctx, spanName, opts...)
+}
+
+func (cnxn *ConnectionImplBase) SetErrorOnSpan(span trace.Span, err error) bool {
```

Comment:
Does this really need to be part of the interface vs. having a utility method?

Reply:
Moved to `internal`